test/e2e/specs/customer_admin.go

```diff
@@ -240,5 +241,26 @@ var _ = Describe("Openshift on Azure customer-admin e2e tests [CustomerAdmin][Fa
 		Expect(err).ToNot(HaveOccurred())
 	})
 
+	It("should sync AAD admin group", func() {
+		syncSecret, err := cli.Client.Admin.CoreV1.Secrets("openshift-infra").Get("aad-group-sync-config", metav1.GetOptions{})
+		Expect(err).ToNot(HaveOccurred())
+		idPre := "44e69b"
+		idSuf := "30d7b0"
+		testUser := "testuserdisabled"
+		gid := syncSecret.Data["customerAdminGroupId"]
+		if strings.HasPrefix(string(gid), idPre) && strings.HasSuffix(string(gid), idSuf) {
+			// test the users
+			oca, err := cli.Client.Admin.UserV1.Groups().Get("osa-customer-admins", metav1.GetOptions{})
+			Expect(err).ToNot(HaveOccurred())
+
+			found := false
+			for _, user := range oca.Users {
+				if strings.HasPrefix(user, testUser) {
+					found = true
+				}
+			}
+			Expect(found).To(BeTrue())
```

Comment:
should test that there is one and only one user, and that it matches

Reply:
I will update this.